internal/queryservice/task.go

```diff
@@ -607,20 +607,22 @@ func watchDmChannels(ctx context.Context,
 	if err != nil {
 		return err
 	}
-	channelRequest := datapb.GetInsertChannelsRequest{
-		DbID:         dbID,
+	describeCollectionRequest := &milvuspb.DescribeCollectionRequest{
+		Base: &commonpb.MsgBase{
+			MsgType: commonpb.MsgType_DescribeCollection,
+		},
 		CollectionID: collectionID,
 	}
-	resp, err := dataService.GetInsertChannels(ctx, &channelRequest)
+	resp, err := masterService.DescribeCollection(ctx, describeCollectionRequest)
 	if err != nil {
 		return err
 	}
-	if len(resp.Values) == 0 {
+	if len(resp.VirtualChannelNames) == 0 {
```

Comment:
proxy node insert data into PhysicalChannels, so check the PhysicalChannel would be better